en/docs/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores.md

```diff
@@ -12,6 +12,9 @@ Be sure to go through the [Recommendations for setting up keystores](../../confi
 
 There are two ways to create keystores for WSO2 API Manager. You can either generate a keystore using an already existing public key certificate (CA-signed), or you can create the public key certificate at the time of generating the keystore. See the instructions given below.
 
+!!! note
+    Wildcard certificates are supported to match **any single domain component name only**. Component fragments not being validated is a limitation. For example, `*.a.com` is matched with `name.a.com`. But `n*.a.com` is not supported as a valid component name.
```

Comment:
For example, `name.a.com` is matched with `*.a.com` but not with `n*.a.com`

shall we change it in this way?

Reply:
Changes applied with https://github.com/wso2/docs-apim/pull/4074/commits/db593b75052f4cfb2e1499ea872d6014621e14b8